src/hooks/aelin/useNftCollectionLists.tsx

```diff
@@ -48,49 +47,18 @@ const AELIN_MAINNET_NFT_COLLECTIONS = '/data/nft-metadata/aelin-mainnet-metadata
 const AELIN_OPTIMISM_NFT_COLLECTIONS = '/data/nft-metadata/aelin-optimism-metadata.json'
 const AELIN_ARBITRUM_NFT_COLLECTIONS = '/data/nft-metadata/aelin-arbitrum-metadata.json'
 const AELIN_POLYGON_NFT_COLLECTIONS = '/data/nft-metadata/aelin-polygon-metadata.json'
-const MAINNET_NFT_COLLECTIONS = '/data/nft-metadata/open-sea-mainnet-metadata.json'
-const OPTIMISM_NFT_COLLECTIONS = '/data/nft-metadata/quixotic-metadata.json'
-const ARBITRUM_NFT_COLLECTIONS = '/data/nft-metadata/stratos-metadata.json'
-const POLYGON_NFT_COLLECTIONS = '/data/nft-metadata/open-sea-polygon-metadata.json'
+const MAINNET_NFT_COLLECTIONS = '/data/nft-metadata/mainnet-metadata.json'
+const OPTIMISM_NFT_COLLECTIONS = '/data/nft-metadata/optimism-metadata.json'
+const ARBITRUM_NFT_COLLECTIONS = '/data/nft-metadata/arbitrum-metadata.json'
+const POLYGON_NFT_COLLECTIONS = '/data/nft-metadata/polygon-metadata.json'
```

Comment:
I think these changes need to be reversed; otherwise it won't work because the files won't be found. Also there were some changes in the codebase and i think this approach won't work anymore. We need to import the json file directly.

Reply:
Ref: https://github.com/AelinXYZ/aelin-frontend-v2/commit/991a319451c8609c54d0d1198ac1913f9899a496#diff-64f7cecd0c185afbfb5227845e956e8af238660cce2b96d065bc40669a304314R5